code/modules/nano/interaction/default.dm

```diff
@@ -80,6 +80,13 @@ GLOBAL_DATUM_INIT(default_state, /datum/topic_state/default, new)
 		return STATUS_UPDATE
 
 
+/mob/living/carbon/Xenomorph/default_can_use_topic(src_object)
+	. = shared_nano_interaction(src_object)
+	if(. != STATUS_CLOSE)
+		if(loc)
+			. = min(., loc.contents_nano_distance(src_object, src))
```

Comment:
Can you explain the return type here?

Above it looks like a define to explain whats happening, CLOSE, REFRESH, ETC.
However the 2nd return feels more like a distance variable.

Is that not mixing types?

Reply:
Both of these are a number, which is just an abstraction of the states.